### ISSUE TITLE
 balance not showing last incoming transaction in my balance (transfer between wallets).  I

### ISSUE THREAD

**Issue by bugman68** [Berry-Pool/nami-wallet#204]
balance not showing last incoming transaction in my balance (transfer between wallets).  I saw you had this same issue with someone in #201 The Same fix worked for me, but you need to fix this issue

**Comment by alessandrokonrad** [comment]
Yeah, the problem is that this issue never occurred once with my Nami wallet. I tried to fix it in the past and did various things, but without having this issue it's very hard to debug.
Maybe it is also related to the Blockfrost API, which actually fetches the information from the blockchain.
